modules/machineset-azure-ephemeral-os.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+
+[id="machineset-azure-ephemeral-os_{context}"]
+= Machine sets that deploy machines on Ephemeral OS disks
+
+You can save on costs by creating a machine set running on Azure that deploys machines on Ephemeral OS disks. Ephemeral OS disks use local VM capacity rather than remote Azure Storage. This configuration therefore incurs no additional cost and provides lower latency for reading, writing, and reimaging.
```

Comment:
It seems a bit odd to lead with cost savings. Maybe that's just me. 
_You can create a machine set running on Azure..._